components/hab/src/command/studio.rs

```diff
@@ -111,6 +111,15 @@ mod inner {
             return Ok(());
         }
 
+        // Make sure we have run 'hab setup' already (i.e, .hab folder exists)
+        let cache_dir = cache_path(None);
```

Comment:
Can we do this check somewhere that both Linux and Mac/Windows studio invocations could benefit?  Since we attempt to map the key cache location from the host to the docker-based studio, checking for `hab setup` would be good feedback for new users.